runtime/Go/antlr/internal/testrig/README.adoc

```diff
@@ -1,3 +1,12 @@
-# Test rig for go
+= Test rig for go
 
-This test rig will build and run the grammar file test.g4 with the `input` file and turn on all tracing options.
\ No newline at end of file
+This test rig will build and run the grammar file test.g4 with the `input` file for local testing by maintainers.
+
+You should copy the `dev` brnach version of the antlr4 complete jar in to the antlr directory and you can then run
```

Comment:
Please fix typo:

```suggestion
You should copy the `dev` branch version of the antlr4 complete jar in to the antlr directory and you can then run
```

Reply:
I have removed this internal directory to avoid confusion - I will keep it elsewhere.